bin/moodle-docker-compose.cmd

```diff
@@ -1,5 +1,10 @@
 @ECHO OFF
 
+if "%MOODLE_DOCKER_WWWROOT%"=="" (
```

Comment:
IMHO it should be better to check it via `IF NOT EXIST "%MOODLE_DOCKER_WWWROOT%"`.

HTH,
Matteo

Reply:
Good catch. I've updated the PR.